src/gpodder/util.py

```diff
@@ -595,6 +595,19 @@ def delete_file(filename):
         pass
 
 
+def is_html(text):
```

Comment:
Unless you see something otherwise, I will prefer the [implementation submitted to **podcastparser**](https://github.com/gpodder/podcastparser/pull/12/files#diff-e8db52e77ac4eba974cc9019d91ca9f0R847).  I have not done comparative testing, but just in reading the RegExes that one seems more comprehensive.